src/content/public goods funding.md

```diff
@@ -0,0 +1,287 @@
+
+3. **Negative externalities**: Another downside with democratic funding schemes is the cost borne by the minority. In democratic systems, those who may not necessarily need a public good are forced to pay the same as those who need it the most. For example, Bob (a risk-averse individual) is forced to pay taxes to subsidize medical procedures for Alice (a risk-seeking individual), even though Bob himself might never need such services. 
+
+#### Plutocratic voting {#plutocratic-voting} 
+
+Some funding models use a plutocratic mechanism in which an individual's financial power determines their influence on the process of providing public goods. These systems are typically described as "one dollar, one vote". In theory, plutocratic funding is ideal since it allows for expressing unequal preference for public goods—but it also has problems:
+
+1. **Serving the rich**: Plutocratic voting prioritizes the preferences of the wealthy (who are often the minority) to the detriment of average income-earners that comprise the majority. Thus, public goods that matter to the majority (and have bigger societal benefits) may never receive funding. 
+
+2. **Cost burdens**: It is possible for middle-class citizens to leverage crowdfunding mechanisms in order to compete against wealthy indivdiduals and collectively express their preferences for a public good. But this is considerably unfair since they still have to bear unfavorably higher costs when expressing preferences. 
+
+3. **Negative externalities**: Public goods created via plutocratic mechanisms often produce negative externalities, mostly because of the need to maximize returns on investment. A classic example is the Internet as it exists today: because Internet businesses (e.g., social media platforms) cannot charge consumers directly, they rely on advertising to recoup investment in infrastructure. This is even as reports of the [negative effects of targeted advertising on individuals](https://www.vox.com/platform/amp/the-goods/2020/4/9/21204425/targeted-ads-fertility-eating-disorder-coronavirus) continue to surface. 
+
+## Improving public goods funding with blockchain technology {#blockchain-for-public-goods-funding} 
+
+The problems explained in the previous section lead to "coordination failures" that stall the development of public goods. However, many believe the existence of decentralized blockchain networks, such as Ethereum, can help solve the problem of efficiently funding and developing public goods. 
+
+### Regenerative cryptoeconomics and public goods markets {#regenerative-cryptoeconomics-and-public-goods-markets} 
+
+"Regenerative cryptoeconomics" concerns using blockchain technology to design incentives and mechanisms that support regenerative economic systems. A regenerative economy is one where common resources are replenished regularly to encourage long-term sustainability. 
```

Comment:
The terminology "common resources are replenished regularly" makes it sound like the system requires feeding with capital to keep it going. I see regenerative economics as money systems that have positive impacts on environment and society in ways that also generate new capital assets.

```suggestion
"Regenerative cryptoeconomics" concerns using blockchain technology to design incentives and mechanisms that support regenerative economic systems. A regenerative economy is one that has positive environmental and social externalities that also generate more capital assets. 
```